docs/Image Uploader.md

```diff
@@ -62,3 +62,8 @@ Replace `XXXXXXXXXXXXXXX` with your Base64-encoded user and password
 |Extra headers|`Authorization: Basic XXXXXXXXXXXXXXX`|
 |Image link||
 |Deletion link||
+
+
+### [catbox.moe](https://catbox.moe/)
+It's not possible yet to upload to catbox.moe due to how their upload method is constructed. It requires you to add extra form fields, which unfortunately cannot be done in Chatterino's uploader yet.
```

Comment:
Instead of making a paragraph for each site we don't support, replace this with a header stating "Unsupported sites" or something along those lines, with `catbox.moe` being part of the list with a shorter message stating why it doesn't work. A table would be fine as well.